<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginWIndow</class>
 <widget class="QMainWindow" name="LoginWIndow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>325</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ambulanta</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="dugmePocetniDoktor">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Doktor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dugmePocetniPacijent">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>110</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>19</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pacijent</string>
    </property>
   </widget>
   <widget class="QLabel" name="loginLabel">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Ulogujte se</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
